### PR TITLE
refactored data events and streams into only events, deleted old rout…

### PR DESCRIPTION
**Pull request by brittneypostma**
…es, and fixed others to new data source, also updated css colors for hover on links

Here ya go @ghostdevv

### CODE REVIEW

**Comment by brittneypostma** [comment]
We should have gotten a deploy preview, but GH has been having issues this morning with deploying through git.

File: src/routes/upcoming/+page.svelte [hunk 1]
```diff
@@ -86,7 +88,7 @@
 		</article>
 	{/each}
 
-	{#each data.upcomingEvents as { speakers, title, date, eventUrl }}
+	<!-- {#each data.upcomingEvents as { speakers, title, date, eventUrl }}
```
Comment (ghostdevv) [inline src/routes/upcoming/+page.svelte:91]: Should we just delete the contents of this comment?

File: src/routes/upcoming/+page.server.ts [hunk 2]
```diff
@@ -1,14 +1,13 @@
 import type { PageServerLoad } from './$types';
 import { getUpcoming } from '@data/utils/time';
-import { streams } from '@data/streams';
+// import { streams } from '@data/streams';
 import { events } from '@data/events';
 
 export const load: PageServerLoad = () => {
-	const upcomingStreams = getUpcoming(streams);
+	// const upcomingStreams = getUpcoming(streams);
```
Comment (ghostdevv) [inline src/routes/upcoming/+page.server.ts:7]:
```suggestion
```

File: src/routes/upcoming/+page.server.ts [hunk 3]
```diff
@@ -1,14 +1,13 @@
 import type { PageServerLoad } from './$types';
 import { getUpcoming } from '@data/utils/time';
-import { streams } from '@data/streams';
+// import { streams } from '@data/streams';
```
Comment (ghostdevv) [inline src/routes/upcoming/+page.server.ts:3]:
```suggestion
```

**Review by ghostdevv — CHANGES_REQUESTED** [review]

File: src/routes/upcoming/+page.svelte (hunk 1 above)
Comment (brittneypostma) [reply src/routes/upcoming/+page.svelte:91]: Yeah, must have missed it, I deleted a ton.

**Review by ghostdevv — APPROVED** [review]

**Pull request merged**